superdesk/resource.py

```diff
@@ -160,6 +160,26 @@ def __init__(self, endpoint_name, app, service, endpoint_schema=None):
                 hook_event -= hook_method
                 hook_event += hook_method
 
+        operations_events = {'fetched': 'fetched_resource',
+                             'fetched_item': 'fetched_item',
+                             'create': 'insert',
+                             'created': 'inserted',
+                             'update': 'update',
+                             'updated': 'updated',
+                             'delete': 'delete_item',
+                             'deleted': 'deleted_item'}
+        for method in [method for method in dir(service) if method.startswith('on_')]:
```

Comment:
not sure if this is needed, you could subscribe explicitly in content_types init_app, any benefits to do it like this? also should have some documentation

Reply:
This is a generic implementation for callbacks on other resources operations and seems more elegant to me than creating the subscription in init_app manually.